### DISCUSSION TITLE
Behavior of versions & librairies pages

### DISCUSSION THREAD

**Discussion by Mysterious-Dev** [QuiltMC/developer-wiki#12] [Q&A] [answered]
Hi 👋 ,

I can see that the pages for the versions & libraries were not done. So I wanted to know if these pages would be filled automatically (I mean that the page is created at each build and its content changes according to the avalaibles versions &  libraries) or manually by the users ?

**Mysterious_Dev**

**Comment by OroArmor** [comment, marked as answer]
Yep, the page is built from some templates and markdown files. The markdown files are then turned into HTML, which is placed into the templates, which are then output at the correct paths.

**Reply by Mysterious-Dev** [reply]
Thanks. But my question (if it is not correctly formulated) is to know if the markdown files for the categories will have a dynamic content (i.e. generated automatically with the listing of the subpages) or if it is the user who will modify the file to list the pages he adds ^^.

**Reply by OroArmor** [reply]
The markdown files are 100% manual content. Nothing there is automatically generated, besides code snippets, which are also manual.
